modules/gatsby/src/components/github/github-contributors.jsx

```diff
@@ -28,19 +57,20 @@ export default class GithubContributors extends Component {
 
   render() {
     const {response} = this.state;
```

Comment:
Just having `response` in state looks a bit unfinished. Can't the part of code that loads do the data verification and store `contributors` in state?

Reply:
Probably in another diff.